README.md

```diff
@@ -85,16 +85,20 @@ endpoint=http://code.yourcompany.com/api/v3
 
 When working with Github PR material, we also make available the following list of environment variables in the pipeline.
 
+Note: Their name will vary depending on your material's name. For example, if you name it `my-material`, they will be available under `GO_SCM_MY_MATERIAL_*` prefix.
+
+##### Troubleshooting: Those variables will be seen only if you have managed to setup GH authentication correctly. See [this comment](https://github.com/ashwanthkumar/gocd-build-github-pull-requests/issues/105#issuecomment-428812031) for more info. (Otherwise, you might end up with a situation in which the PR pipeline is triggered, but is missing the env variables.)
+
 | Environment Variable | Description |
 | --- | --- |
-| PR_BRANCH | Pull Request was submitted **from** this branch. Eg. `feature-1` |
-| TARGET_BRANCH | Pull Request was submitted **for** this branch. Eg. `master` |
-| PR_URL | Pull Request URL on the Github |
-| PR_AUTHOR | Name of the user who submitted the Pull Request |
```

Comment:
Somehow, I didn't find that variable in my gocd. 

But now I see in https://github.com/ashwanthkumar/gocd-build-github-pull-requests/blob/master/src/main/java/in/ashwanthkumar/gocd/github/provider/github/GitHubProvider.java#L98 that it should be actually present. 

So now I don't know, if we should include it in README or not? Maybe I can add it but with an additional note "might not work, see this comment".

Reply:
Sounds good. Not sure if it is got to do with the Github Client we're using. Can you please raise it as an issue? Me or someone else might look into it.